src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -90,6 +90,11 @@ void Terminal::UpdateSettings(ICoreSettings settings)
     _autoMarkPrompts = settings.AutoMarkPrompts();
     _rainbowSuggestions = settings.RainbowSuggestions();
 
+    if (_stateMachine)
+    {
+        SetVtChecksumReportSupport(settings.AllowVtChecksumReport());
+    }

```

Comment:
Now that I removed that DECKPAM setting, I want double check. Is this going to work fine, or will this also affected by ConPTY shenanigans? 😅

Reply:
I think this should be fine, because the VT query responses are now handled by the connected conpty terminal rather than conhost.